webapp/components/wpt-processor.js

```diff
@@ -73,23 +74,27 @@ class WPTProcessor extends LoadingState(PolymerElement) {
         <vaadin-grid-column auto-width header="GitHub Check Run" hidden>
           <template>[[item.check_run_id]]</template>
         </vaadin-grid-column>
-        <vaadin-grid-column auto-width header="SHA">
-          <template>[[item.full_revision_hash]]</template>
+        <vaadin-grid-column auto-width header="Product">
+          <template>[[_product(item)]]</template>
+        </vaadin-grid-column>
+        <!-- Only show the prefix of a SHA, but still allow finding in page. -->
```

Comment:
Is showing the prefix enforced via the explicit width? That wasn't 100% clear to me, might suggest:

`<!-- Set explicit width to only show the prefix of a SHA, but still allow find-in-page. -->`